lib/node_modules/@stdlib/assert/is-same-typed-array-like/lib/main.js

```diff
@@ -0,0 +1,64 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var isTypedArrayLike = require( '@stdlib/assert/is-typed-array-like' );
+var hasSameValues = require( '@stdlib/array/base/assert/has-same-values' );
+
+
+// MAIN //
+
+/**
+* Tests if two arguments are both typed-array-like objects and have the same values.
+*
+* @param {*} v1 - first value
+* @param {*} v2 - second value
+* @returns {boolean} boolean result
+*
+* @example
+* var Int8Array = require( '@stdlib/array/int8' );
+* var Int16Array = require( '@stdlib/array/int16' );
```

Comment:
Put one empty line after Line 38 and Line 47 to maintain consistency in example with other files.